fgd/point/prop/prop_static.fgd

```diff
@@ -110,10 +108,7 @@
 		"If 'Yes', the fade distance is the size of the object onscreen, in pixels."
 	fademindist(float) : "Start Fade Dist/Pixels" : -1 : "Distance at which the prop starts to fade (<0 = use fademaxdist). If 'Screen Space Fade' is selected, this represents the number of pixels wide covered by the prop when it starts to fade."
 	fademaxdist(float) : "End Fade Dist/Pixels" : 0 : "Maximum distance at which the prop is visible (0 = don't fade out). If 'Screen Space Fade' is selected, this represents the *minimum* number of pixels wide covered by the prop when it fades."
-	fadescale(float) : "Fade Scale" : 1 : "If you specify a fade in the worldspawn, " +
-		"or if the engine is running under dx7 [hl2/ep1/portal] or dx8 [ep2/tf], " +
-		"then the engine will forcibly fade out props even if fademindist/fademaxdist " +
-		"isn't specified. " +
+	fadescale(float) : "Fade Scale" : 1 : "If you specify a fade in the worldspawn. " +
```

Comment:
This comment seems malformed. I think the correct string to use is:
```
If you specify a fade in the worldspawn, then the engine will forcibly fade out props even if fademindist/fademaxdist isn't specified.
```

Reply:
woops! stupid mistake